submissions/akinbobolaFaith/dom/signUp.js

```diff
@@ -0,0 +1,122 @@
+let signupForm = document.querySelector('.information'),
+    firstName = document.getElementById('fname'),
+    lastName = document.getElementById('lname'),
+    phoneNumber = document.getElementById('pnumber'),
+    email = document.getElementById('email'),
+    password = document.getElementById('password'),
+    confirmPassword = document.getElementById('cpassword')
+;
+
+function important (field, event) {
```

Comment:
Merge the function important to first name, last name and other functions @faithakinbobola